zulipterminal/ui_tools/boxes.py

```diff
@@ -355,12 +391,14 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     )
             if success:
                 self.msg_write_box.edit_text = ''
+                self.send_stop_typing_status()
                 if self.msg_edit_id:
                     self.msg_edit_id = None
                     self.keypress(size, 'esc')
         elif is_command_key('GO_BACK', key):
             self.msg_edit_id = None
             self.msg_body_edit_enabled = True
+            self.send_stop_typing_status()
```

Comment:
I can do this inside the method, using the same check that we have for sending a message, checking for `self.to_write_box`'s contents. I can mention that it is meant only for PMs in a comment in the method's definition, we might not need to rename it as we only show typing in PM narrows.

Reply:
I'll use one inside the method, along with a comment mentioning that it is only for PM narrows. I don't think a rename is necessary, because technically, we could send the stop status for streams using that same function, should we add that feature in the future.